.env

```diff
@@ -1 +0,0 @@
-DATABASE_URL = postgres://postgres:password@localhost:5432/odi_development
```

Comment:
Hi, Arya! Please prevent the following action from rehappening as it might leak the credentials we use to run our application. When those data are leaked, there are no other options than to purchase a new service which might be hassle given our budget situation.